# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gobject-introspection/hotfix-no-tests.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gobject-introspection-1.80.1/meson.build.vanilla	2024-04-09 19:03:45.454000000 +0300
+++ gobject-introspection-1.80.1/meson.build	2024-04-09 19:03:16.442000000 +0300
@@ -249,7 +249,7 @@
 # The tests will also run, which is not possible if they
 # were built for a different architecture.
 if not meson.is_cross_build()
-  subdir('tests')
+  #subdir('tests')
 endif
 
 install_data('Makefile.introspection', install_dir: join_paths(get_option('datadir'), 'gobject-introspection-1.0'))
